tests/2017-04-01/Microsoft.ServiceBus.tests.json

```diff
@@ -291,6 +291,34 @@
           }
         }
       }
+    },
+    {
+      "name": "migration - invalidType",
+      "expectedErrors": [
+        {
+          "message": "Missing required property: name",
+          "dataPath": "/",
+          "schemaPath": "/required/0"
+        },
+        {
+          "message": "No enum match for: \"Microsoft.ServiceBus/invalidType\"",
+          "dataPath": "/type",
+          "schemaPath": "/properties/type/type"
+        },
+        {
+          "message": "No enum match for: \"2015-10-12\"",
+          "dataPath": "/apiVersion",
+          "schemaPath": "/properties/apiVersion/type"
+        }
+      ],
+      "definition": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.ServiceBus.json#/resourceDefinitions/namespaces_migrationConfigurations",
+      "json": {
+        "type": "Microsoft.ServiceBus/invalidType",
+        "apiVersion": "2015-10-12",
+        "configName": "$default",
+        "properties": {
+        }
+      }
```

Comment:
can we add another test case with a valid payload for  "Microsoft.ServiceBus/namespaces/migrationConfigurations"?

Reply:
resolved, added a valid playload test.